integration_tests/src/go-tests/without-ddlambda/main.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"context"
+
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/lambda"
+)
+
+type testResponse struct {
+	StatusCode int    `json:"statusCode"`
+	Body       string `json:"body"`
+}
+
+func PingHandler(ctx context.Context, ev events.APIGatewayProxyRequest) (testResponse, error) {
```

Comment:
Let's add a log line here to make sure logging works without the ddlambda library

Reply:
Actually, I think you could remove the "logs test" and move the log lines into the with-dd-lambda and without-dd-lambda tests. That way we cover logs in both cases.